common/api-review/storage.api.md

```diff
@@ -18,6 +18,11 @@ import { Unsubscribe } from '@firebase/util';
 // @public
 export function connectStorageEmulator(storage: FirebaseStorage, host: string, port: number): void;
 
+// Warning: (ae-forgotten-export) The symbol "StringData" needs to be exported by the entry point index.d.ts

```

Comment:
Do we need to fix these warnings?

Reply:
no, because these are from internal APIs.